<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>WelcomeWidget</class>
    <widget class="QWidget" name="WelcomeWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1029</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Welcome</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QLabel" name="welcome_label">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>24</pointsize>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">color: rgb(255, 255, 255);
                            background-color: rgb(94, 94, 94);
                        </string>
                    </property>
                    <property name="text">
                        <string>COCR: a handwriting OCSR toolset</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" name="gridLayout" rowstretch="618,1000" columnstretch="1000,618">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item row="0" column="0">
                        <layout class="QVBoxLayout" name="intro_layout">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="intro_label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>23</pointsize>
                                        </font>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(226, 226, 226);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>introduction</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="intro_edit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="inputMethodHints">
                                        <set>Qt::ImhNone</set>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                    <property name="html">
                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                            &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;
                                            /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                            p, li { white-space: pre-wrap; }
                                            &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun';
                                            font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                        </string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="0" column="1" rowspan="2">
                        <layout class="QVBoxLayout" name="agree_layout">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="agree_label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>23</pointsize>
                                        </font>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(255, 255, 255);
                                            background-color: rgb(0, 0, 0);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>agreement</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextBrowser" name="agree_edit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="inputMethodHints">
                                        <set>Qt::ImhNone</set>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                                <widget class="QCheckBox" name="agree_box">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>16</pointsize>
                                        </font>
                                    </property>
                                    <property name="layoutDirection">
                                        <enum>Qt::LeftToRight</enum>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string>i agree</string>
                                    </property>
                                    <property name="icon">
                                        <iconset resource="../../../../assets/COCR_res.qrc">
                                            <normaloff>:/images/home.png</normaloff>:/images/home.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>32</width>
                                            <height>32</height>
                                        </size>
                                    </property>
                                    <property name="tristate">
                                        <bool>false</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="1" column="0">
                        <layout class="QVBoxLayout" name="guide_layout">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="guide_label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>23</pointsize>
                                        </font>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(226, 226, 226);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>guide</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="guide_edit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="inputMethodHints">
                                        <set>Qt::ImhNone</set>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                    <property name="html">
                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                            &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot;
                                            /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                            p, li { white-space: pre-wrap; }
                                            &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun';
                                            font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;
                                            &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                            margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                        </string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../../../../assets/COCR_res.qrc"/>
    </resources>
    <connections/>
</ui>
